frontend-react/src/hooks/UseOktaMemberships.ts

```diff
@@ -80,16 +93,16 @@ export const getSettingsFromOrganization = (
 ): MembershipSettings => {
     const parsedName = parseOrgName(org);
     const memberType = getTypeOfGroup(org);
-    let senderName = extractSenderName(org);
+    let service = extractSenderName(org);
 
-    if (memberType === MemberType.SENDER && !senderName) {
-        senderName = "default";
+    if (memberType === MemberType.SENDER && !service) {
```

Comment:
Hmm, woops. Missed this one. This is using the embedded service in the Okta name (like `DHSender_ignore.full-elr`). Will need to look into this more